lib/Base/Tensor.cpp

```diff
@@ -282,6 +282,48 @@ static void transposeSelectImpl(const Handle<ElemTy> &src, Handle<ElemTy> &dest,
     transposeGenericImpl(src, dest, srcCoor, destCoor, shuffle);
   }
 }
+
+template <class ElemTy>
+static bool isTiledAlongAxisImpl(const Tensor *tensor, size_t axis,
```

Comment:
I wonder if this could/should be used in place of https://github.com/pytorch/glow/blob/0d4af705d00aec7e7bd41fa2e3ec5478247afd0f/lib/Optimizer/GraphOptimizer/GraphOptimizer.cpp#L1803

CC: @vuzelac-cadence

Reply:
From what I see the `isConstBroadcasted` also extracts the repeating slice which means is a different functionality.
Also next time please do not allow people to write utility functions which are so limited (this functions assumes only 4D). Anybody could spend an extra 10% effort to make an utility function which is generic enough to be used by anybody in more larger contexts.